lib/modules/manager/maven/update.ts

```diff
@@ -141,3 +177,24 @@ function isSnapshot(
   const lastPart = prerelease?.at(-1);
   return is.string(lastPart) && lastPart.endsWith('SNAPSHOT');
 }
+
+function updateValue(
+  content: string,
+  nodeName: string,
+  oldValue: string,
+  newValue: string,
+): string {
+  const elementStart = content.indexOf('<' + nodeName);
+  const start = content.slice(elementStart).indexOf('>') + elementStart + 1;
+  const end = content.slice(start).indexOf('</' + nodeName) + start;
```

Comment:
This can be made simpler:
```suggestion
  const start = content.indexOf('>', elementStart) + 1;
  const end = content.indexOf('</' + nodeName, start);
```